packages/remix-server-runtime/reexport.ts

```diff
@@ -0,0 +1,49 @@
+export type {
+  HandleDataRequestFunction,
+  HandleDocumentRequestFunction,
+  ServerBuild,
+  ServerEntryModule,
+} from "./build";
+
+export type {
+  Cookie,
+  CookieOptions,
+  CookieParseOptions,
+  CookieSerializeOptions,
+  CookieSignatureOptions,
+} from "./cookies";
+
+export type { AppLoadContext, AppData } from "./data";
+
+export type { EntryContext } from "./entry";
+
+export type {
+  HtmlLinkDescriptor,
+  LinkDescriptor,
+  PageLinkDescriptor,
+} from "./links";
+
+export type { ServerPlatform } from "./platform";
```

Comment:
Pretty sure we don't need this anymore with this work. See my other comment about the platform arg to createRequestHandler.